crates/notary/server/config/config.yaml

```diff
@@ -12,21 +12,21 @@ server:
     <a href="/healthcheck">health check</a> - <a href="/info">info</a><br/>
 
 notarization:
-  max-sent-data: 4096
-  max-recv-data: 16384
+  max_sent_data: 4096
+  max_recv_data: 16384
 
 tls:
   enabled: true
-  private-key-pem-path: "./fixture/tls/notary.key"
-  certificate-pem-path: "./fixture/tls/notary.crt"
+  private_key_pem_path: "../fixture/tls/notary.key"
```

Comment:
the path was correct, why did you change it?

Reply:
on my end the path didn't load

Reply:
i cloned ur branch and did the following; and it worked — are you running from `crate/notary/server`?

![image](https://github.com/user-attachments/assets/5343c2dd-108d-4702-8eb1-b2c96e6add6b)
![image](https://github.com/user-attachments/assets/ecddc45c-f216-42ce-a907-63cd588a1e35)